README.md

```diff
@@ -56,8 +56,15 @@ The main advantage of a free text search is that you can order the results in or
 ## Using CHS docker
```

Comment:
Is this section meant to be a sub-level of the example curl command? i.e., is it meant to be `### Using CHS docker` instead of `## Using CHS docker`